lib/with_advisory_lock/concern.rb

```diff
@@ -33,6 +33,10 @@ def current_advisory_lock
         lock_stack_key && lock_stack_key[0]
       end
 
+      def current_advisory_locks
+        WithAdvisoryLock::Base.lock_stack.map(&:name)
```

Comment:
As an alternative, this would work too

`WithAdvisoryLock::Base.lock_stack.map { |lock| lock[0] }`